daemon/openrazer_daemon/hardware/mouse.py

```diff
@@ -88,7 +88,32 @@ class RazerMambaTE(__RazerDeviceBrightnessSuspend):
     METHODS = ['get_device_type_mouse', 'set_wave_effect',
                'set_static_effect', 'set_spectrum_effect', 'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect',
                'set_breath_single_effect', 'set_breath_dual_effect', 'set_custom_effect', 'set_key_row', 'max_dpi',
-               'get_dpi_xy', 'set_dpi_xy']
+               'get_dpi_xy', 'set_dpi_xy', 'get_poll_rate', 'set_poll_rate']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/606/606_mambate_500x500.png"
+
+    # Deprecated - RAZER_URLS be removed in future.
+    RAZER_URLS = {
+        "top_img": "https://assets.razerzone.com/eeimages/support/products/606/606_mambate_500x500.png",
+        "side_img": "https://assets.razerzone.com/eeimages/products/22294/mambategallery-800x800-5.png",
+        "perspective_img": "https://assets.razerzone.com/eeimages/products/22294/mambategallery-800x800-2.png"
+    }
+
+    DPI_MAX = 16000
+
+
+class RazerLanceheadTE(__RazerDeviceBrightnessSuspend):
+    """
+    Class for the Razer Lancehead Tournament Edition
+    """
+    USB_VID = 0x1532
+    USB_PID = 0x0060
+    HAS_MATRIX = True
+    MATRIX_DIMS = [1, 16]
+    METHODS = ['get_device_type_mouse', 'set_wave_effect',
+               'set_static_effect', 'set_spectrum_effect', 'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect',
+               'set_breath_single_effect', 'set_breath_dual_effect', 'set_custom_effect', 'set_key_row', 'max_dpi',
+               'get_dpi_xy', 'set_dpi_xy', 'get_poll_rate', 'set_poll_rate']
 
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/606/606_mambate_500x500.png"
```

Comment:
Please find an image for your device here

edit: you seem to have pasted the urls for this device down below. See the diff for details

Reply:
I am unable to find proper side/perspective images...

Reply:
Wait, thanks for pointing out. Will fix.